src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -1104,6 +1104,22 @@ g.test('undefined_params')
     });
   });
 
+g.test('coppy_from_depth_aspect')
+  .desc(
+    `
+  Validate the correctness of copyTextureToBuffer() with depth aspect.
```

Comment:
Same suggestion as for stencil to try to merge the test in the existing parametrization to take advantage of the amazing coverage.

Reply:
The plan on how the test would work LGTM though.